modules/siddhi-core/src/main/java/org/wso2/siddhi/core/util/snapshot/SnapshotService.java

```diff
@@ -39,10 +39,7 @@ public void addSnapshotable(Snapshotable snapshotable) {
 
     public byte[] snapshot() {
         HashMap<String, Object[]> snapshots = new HashMap<String, Object[]>(snapshotableList.size());
-
-        if (log.isDebugEnabled()) {
-            log.debug("Taking snapshot ...");
-        }
+        log.info("Taking snapshot ...");
```

Comment:
Thanks again @nirmal070125 , As per the talk with suho, syncing a snapshot is vital in HA and we need to know when it starts to sync and when the process ends even though it logs few times.